### ISSUE TITLE
同志们，YOLO v10究极踩坑。

### ISSUE THREAD

**Issue by DrinkOcean** [THU-MIG/yolov10#381]
如果你是从 parse_model 构建模型的话，那么，要加上：
    for m in model.modules():
        t = type(m)
        if t is nn.BatchNorm2d:
            m.eps = 1e-3
            m.momentum = 0.03
推理的时候，更得加上，否则预测出来的 bbox 一点都不对。
(找这个bug，找了2天，我没开玩笑！)，tnnd，累了，非常累。
